brave/brave-core
# Sign transactions with Trezor

Resolves https://github.com/brave/brave-browser/issues/19029

- Added signing Transactions with Trezor
- Split Trezor and Ledger transactions signing for separate functions
- Open Trezor library popup in new window
- Keep wallet panel opened until Trezor popup closed
- Close all popups from wallet panel if the panel is closed.

https://user-images.githubusercontent.com/2965009/139705279-9133d472-485d-4b2e-a07c-4c77cb3e9ba0.mp4

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
- Import Ganache account
- Import Hardware account
- Move money from Ganache to Hardware and back
- Check for both Ledger and Trezor devices
